### PR TITLE
feat: add documentation for new terraform plugin

### PR DESCRIPTION
**Pull request by kneal**

### CODE REVIEW

**Review by meinenec — DISMISSED** [review]

**Comment by wass3r** [comment]
probably want to wait until https://github.com/go-vela/vela-terraform/pull/15 is merged and tagged before merging this in

**Comment by kneal** [comment]
@wass3r Good call converting to draft

File: content/plugins/registry/terraform.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+## Usage
+
+**NOTE: by default Terraform runs in the current directory. Use `directory: path/to/tf/files` to point Terraform at a file or files.**
+
+Sample of adding init options to Terraform configuration:
+
+```yaml
+- name: apply
+  image: target/vela-terraform:latest
```
Comment (jbrockopp) [inline content/plugins/registry/terraform.md]: Thoughts on updating the `image` from `latest` to a tagged release of the plugin?

Comment (kneal) [reply content/plugins/registry/terraform.md]: Yeah, I will add now that the repository is tagged. I was doing further testing before tagging but feeling good now. Also, one last docs PR I missed will need to be propagated here.

**Review by jbrockopp — APPROVED** [review]
LGTM

**Review by wass3r — APPROVED** [review]

**Pull request merged**